index-about.html

```diff
@@ -0,0 +1,103 @@
+          <h4 class="text-center" id="h4">"Hello! Lets start the journey."</h4>
+          <h1 class="text-center" id="h1">EPIC ENDGAME PROGRAM SERIES 2022</h1>
+        </div>
+        <div class="p-4 mb-2 bg-light text-dark" >
+          <p class="text-center">Let's really CRUSH this body workout together!These workouts are going to challenge your body endurance and your strength. Join my program see result in less that 3 weeks.</p>
+        </div>
+       <div class=" p-4">
+         <p class="text-center m-0">PLease contact us per Email for any other questions about the program of 2022!</p>
+         <p class="text-center text-decoration-underline fw-bold">suzana.marsela@gmail.com</p>
+       </div>
+      </div>
+    </div>
+    <div class="wrapper2 border p-4">
+     <h4 class="text-dark text-center p-3">Creative Commons <br>Global Exercise 2015 Logo </h4>
+     <div class="line "></div>
+     <p class="text-center p-3" >The logo of this company was decided on 2015 throught a discussion between the members on 7 July.</p>
+     <img class="w-50 img-responsive center-block d-block mx-auto" src="images/logos.png" alt="">
+    </div>
+
+    <div class="wrapper3 border">
+      <h3 class="text">See the past event this year</h3>
+      <div class="line"></div>
+      <p class="text-center p-3" >Check these two events of Global Exercise happening in USA and Irland.</p>
+      <div class="red-wrap">
+        <div class=" container p-0 pt-4">
+          <div class="eventpic1 d-flex  mx-auto justify-content-center align-content-center align-items-center m-0 ">
+            <div class="">
+              <h3 class="text-center text-light m-0 p-0">2017</h3> <br>
+              <p class="text-center text-light">Lorem ipsum dolor sit amet, consectetur</p>
+            </div>
+          </div>
+        </div>
+       <div class=" container p-0 pt-4">
+        <div class="eventpic2 d-flex  mx-auto justify-content-center align-content-center align-items-center m-0 ">
+          <div class="">
+            <h3 class="text-center text-light m-0 p-0">2020</h3> <br>
+            <p class="text-center text-light">Lorem ipsum dolor sit amet, consectetur</p>
+          </div>
+        </div>
+      </div>
+      </div>
+
+
+    </div>
+<div class=" wrapper4">
+  <h5 class="text-light text-center">Partner</h5>
+  <div class="line"></div>
+  <section class="d-flex flex-row  justify-content-center  p-4">
+    <p class=" text-light p-3">Google</p>
+    <p class=" text-light p-3">AIRBNB</p>
+    <p class=" text-light p-3">Mozzilla</p>
+    <p class=" text-light p-3">Twitter</p>
+    <p class=" text-light p-3">Youtube</p>
+  </section>
+</div>
+<div class="wrapper5 ">
+  <div class="p-2 border">
+    <div class="d-flex flex-row justify-content-center align-items-center align-content-center p-4">
+      <img class="w-25 p-0" src="images/logos.png" alt="">
+      <p class="font-weight-light">2015 Gym Workouts. Some Rights Reserved.</p>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] I think it would be better to use HTML5 semantic tag `<footer>` over `divs` tags , To follow [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). 👌 
> instead of `<div class="wrapper5 ">` I suggest using `<footer class="wrapper5 ">`

- [ ] Kindly, make sure you implement this change on the home page as well. 👍

- [ ]  Also, according to the original design, The footer section in the about page (with a dark background-color) on the `desktop version`, so kindly change the footer background-color to black (dark) on the desktop version 💯👌
remember, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including colors💯

Colors.


| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/181511656-fee178a1-d8d2-49b0-bd61-ad0968f8ab8a.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/181510625-de696a22-c2cb-49e3-83b2-0df95677c02a.png" width="500" /> |